migrations/20220324080020-add-place-comment.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.createTable(
+        'comments',
+        {
+          id: {
+            type: Sequelize.UUID,
+            primaryKey: true,
+            allowNull: false,
+            unique: true,
+            autoIncrement: false,
+            defaultValue: Sequelize.fn('uuid_generate_v4'),
+          },
+          title: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          message: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          links: {
+            type: Sequelize.ARRAY(Sequelize.TEXT),
+            allowNull: true,
+          },
+          place_id: {
+            type: Sequelize.UUID,
+            allowNull: true,
+            references: {
+              model: 'places',
+              key: 'id',
+            },
+            onUpdate: 'set null',
```

Comment:
is this correct? Do we want to have orphaned comments?

Reply:
Yup, that makes more sense.